pom.xml

```diff
@@ -8,9 +8,81 @@
     <artifactId>Dmitrii_Pikhurov</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <testng.version>7.4.0</testng.version>
+        <assertj-core.version>3.20.2</assertj-core.version>
+        <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+        <maven-surefire-plugin.suiteXmlFile>${project.build.testOutputDirectory}/testng/calculator.homework1.addition.subtraction.xml
+        </maven-surefire-plugin.suiteXmlFile>
+        <calculator.version>1</calculator.version>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven-surefire-plugin.version}</version>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>
+                            ${maven-surefire-plugin.suiteXmlFile}
+                        </suiteXmlFile>
+                    </suiteXmlFiles>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>calculator-homework1-addition-subtraction-test</id>
+            <properties>
+                <maven-surefire-plugin.suiteXmlFile>${project.build.testOutputDirectory}/testng/calculator.homework1.addition.subtraction.xml
+                </maven-surefire-plugin.suiteXmlFile>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>calculator-homework1-multiply-division-test</id>
+            <properties>
+                <maven-surefire-plugin.suiteXmlFile>${project.build.testOutputDirectory}/testng/calculator.homework1.multiply.division.xml
+                </maven-surefire-plugin.suiteXmlFile>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>calculator-homework1-overall-test</id>
+            <properties>
+                <maven-surefire-plugin.suiteXmlFile>${project.build.testOutputDirectory}/testng/calculator.homework1.overall.xml
+                </maven-surefire-plugin.suiteXmlFile>
+            </properties>
+        </profile>
+
+    </profiles>
+
+    <dependencies>
+        <dependency>
+            <groupId>com</groupId>
+            <artifactId>calculator</artifactId>
+            <version>${calculator.version}</version>
```

Comment:
Where is the calculator placed? I can't run your homework
Why did you delete .gitgnore file?

Reply:
Calculator placed in "\%userprofile\%\.m2\repository\com\calculator\1\calculator-1.jar", in standard maven local repository folder. As I now understand it is necessary to place jar file in project folder itself? I will do it right right now.

I delete .gitignore because in 4 paragraph of "Git_workflow_description" was written:
".gitignore file should include ignored folders and files that do not contain source code"
.gitignore does not contain source code itself.